src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -57,18 +58,22 @@ export default function NavigationBar() {
             </Menu>
             <hr style={{ margin: "0" }} />
             <Menu className={`w-full text-lg ${styles.customizedMenu}`}>
-              <MenuItem
-                className="flex w-full text-start"
-                icon={<FontAwesomeIcon icon={faHouse} />}
-              >
-                Dashboard
-              </MenuItem>
-              <MenuItem
-                className="flex w-full text-start"
-                icon={<FontAwesomeIcon icon={faUsers} />}
-              >
-                Members
-              </MenuItem>
+              <Link to={`dashboard`}>
+                <MenuItem
+                  className="flex w-full text-start"
+                  icon={<FontAwesomeIcon icon={faHouse} />}
+                >
+                  Dashboard
+                </MenuItem>
+              </Link>
+              <Link to={`members`}>
+                <MenuItem
+                  className="flex w-full text-start"
+                  icon={<FontAwesomeIcon icon={faUsers} />}
+                >
+                  Members
+                </MenuItem>
+              </Link>
```

Comment:
Could we do this in a for loop to remove duplicate code?
Like have an array objects where each object would contain the date for each menu item and even handle submenus as well.

Reply:
Resolved